src/RA_AchievementPopup.h

```diff
@@ -43,42 +42,42 @@ class MessagePopup
     {
     }
 
-public:
+    ~MessagePopup() = default;
+    MessagePopup(const MessagePopup&) = delete;
+    MessagePopup(MessagePopup&&) = default;
```

Comment:
This one should be `noexcept`. I think you should `delete` the copy assignment since copying is deleting and the move assignment operator since all members are `const`.